src/main/webapp/app/exercises/programming/manage/update/update-components/language/programming-exercise-language.component.html

```diff
@@ -0,0 +1,240 @@
+<section>
```

Comment:
Because the folder structure was adjusted aswell git thinks this would be a completely new file - however, only `@if` conditions were added to make sure that only the fields are displayed that are relevant for the current edit mode

Reply:
Compare to src/main/webapp/app/exercises/programming/manage/update/update-components/programming-exercise-language.component.html